warehouse/static/js/warehouse/controllers/statuspage_controller.js

```diff
@@ -0,0 +1,258 @@
+    {
+      "created_at": "2014-05-14T14:24:40.430-06:00",
+      "id": "w1zdr745wmfy",
+      "impact": "none",
+      "incident_updates": [
+        {
+          "body": "Our data center has informed us that they will be performing routine network maintenance. No interruption in service is expected. Any issues during this maintenance should be directed to our support center",
+          "created_at": "2014-05-14T14:24:41.913-06:00",
+          "display_at": "2014-05-14T14:24:41.913-06:00",
+          "id": "qq0vx910b3qj",
+          "incident_id": "w1zdr745wmfy",
+          "status": "scheduled",
+          "updated_at": "2014-05-14T14:24:41.913-06:00"
+        }
+      ],
+      "monitoring_at": null,
+      "name": "Network Maintenance (No Interruption Expected)",
+      "page_id": "2p66nmmycsj3",
+      "resolved_at": null,
+      "scheduled_for": "2014-05-17T22:00:00.000-06:00",
+      "scheduled_until": "2014-05-17T23:30:00.000-06:00",
+      "shortlink": "http://stspg.dev:5000/Q0F",
+      "status": "scheduled",
+      "updated_at": "2014-05-14T14:24:41.918-06:00"
+    },
+    {
+      "created_at": "2014-05-14T14:27:17.303-06:00",
+      "id": "k7mf5z1gz05c",
+      "impact": "minor",
+      "incident_updates": [
+        {
+          "body": "Scheduled maintenance is currently in progress. We will provide updates as necessary.",
+          "created_at": "2014-05-14T14:34:20.036-06:00",
+          "display_at": "2014-05-14T14:34:20.036-06:00",
+          "id": "drs62w8df6fs",
+          "incident_id": "k7mf5z1gz05c",
+          "status": "in_progress",
+          "updated_at": "2014-05-14T14:34:20.036-06:00"
+        },
+        {
+          "body": "We will be performing rolling upgrades to our web tier with a new kernel version so that Heartbleed will stop making us lose sleep at night. Increased load and latency is expected, but the app should still function appropriately. We will provide updates every 30 minutes with progress of the reboots.",
+          "created_at": "2014-05-14T14:27:18.845-06:00",
+          "display_at": "2014-05-14T14:27:18.845-06:00",
+          "id": "z40y7398jqxc",
+          "incident_id": "k7mf5z1gz05c",
+          "status": "scheduled",
+          "updated_at": "2014-05-14T14:27:18.845-06:00"
+        }
+      ],
+      "monitoring_at": null,
+      "name": "Web Tier Recycle",
+      "page_id": "2p66nmmycsj3",
+      "resolved_at": null,
+      "scheduled_for": "2014-05-14T14:30:00.000-06:00",
+      "scheduled_until": "2014-05-14T16:30:00.000-06:00",
+      "shortlink": "http://stspg.dev:5000/Q0G",
+      "status": "in_progress",
+      "updated_at": "2014-05-14T14:35:12.258-06:00"
+    }
+  ]
+};
+
+const mockFetch = function(endpoint){
+  if (endpoint.indexOf('status.json') !== -1) {
+    return new Promise((resolve, reject) => resolve(mockStatusResponse));
+  } else if (endpoint.indexOf('summary.json') !== -1) {
+    return new Promise((resolve, reject) => resolve(mockSummaryResponse));
+  }
+};
```

Comment:
I assume you're using these for testing locally, maybe we could move these to a comment in the PR? or maybe a patch we can then apply? just thinking of reducing the noise in the PR